File: 5KYU/Number of trailing zeros of N!/solution.cs
using System;
public static class Kata 
{
	public static int TrailingZeros(int n)
	{  		
		var result = 0;
		for (var i = 5; n / i >= 1; i *= 5)
			result += n / i;
		return result;
	}
}

File: 5KYU/Number of trailing zeros of N!/solution.rs
fn zeros(n: u64) -> u64 {
    let mut result = 0;
    let mut done = false;
    let mut i = 5;
    while !done {
        result = result + (n / i);
        i *= 5; 
        if n / i < 1 {
            done = true;
        }
    }
    return result;
}